src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml

```diff
@@ -24,9 +24,12 @@ config:
 
 tests:
     - label: "Commission DUT to TH"
-      verification: |
-
-      disabled: true
+      cluster: "DelayCommands"
```

Comment:
To actually automate this, it needs to move out of the manual tests section into the automated tests one in src/app/tests/suites/tests.js....

Reply:
looks like it is already there under "DeviceManagement".